server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultConnectOptionsContext.java

```diff
@@ -56,11 +56,12 @@
 
     private static final long DEFAULT_WS_INACTIVITY_TIMEOUT_MILLIS = 0L;
     private static final int DEFAULT_HTTP_KEEPALIVE_TIMEOUT = 30; //seconds
+    private static final int DEFAULT_HTTP_KEEPALIVE_MAX_CONNECTIONS = 5;
```

Comment:
gateway.server doesn't have any dependency on http resource address. So, I guess gateway.server would just pass the config (if it is specified) to resource address. Then resource address will deal with default value. That way there is only one place we deal with the default values. So removing the constant here.